src/sources/mod.rs

```diff
@@ -11,7 +11,7 @@ pub mod generator;
 pub mod http;
 #[cfg(feature = "sources-internal_metrics")]
 pub mod internal_metrics;
-#[cfg(feature = "sources-journald")]
+#[cfg(all(feature = "sources-journald", feature = "unix"))]
```

Comment:
Unfortunately, this not works :disappointed:

Reply:
How not? Is it because of `mac`?

Reply:
I'm not sure why, but `cargo check --tests --no-default-features --features sources-journald` not react on `sources/journald.rs` changes. I have the same issue in #3298.

`cfg` in two lines works, I'll fix it in #3298.
```rust
#[cfg(unix)]
#[cfg(feature = "sources-journald")]
```

Reply:
Actually, this can be written in 1 line:
```rust
#[cfg(unix, feature = "sources-journald")]
```
[examples in reference](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg-attribute)

Reply:
That's right, we have a `unix` feature but it's not used like this, so just using `unix` is correct. Good catch. This is also used in `internal_events/mod.rs` so I would suggest you open a PR that fixes this.